changelog/8.6.0.yaml

```diff
@@ -0,0 +1,283 @@
+version: 8.6.0
+entries:
+    - kind: feature
+      summary: Only support elasticsearch as an output for the beta synthetics integration.
+      description: ""
+      component: synthetics-integration
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/1491
+      issue: []
+      timestamp: 1665780486
+      file:
+        name: 1665780486-heartbeat-es-output-only.yaml
+        checksum: 361f993ba3220aef531f7949308e61d2cbf82e03
+    - kind: feature
+      summary: Service runtime for V2
+      description: Service runtime for V2, tailored specifically for Endpoint service.
+      component: ""
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/1529
+      issue:
+        - https://github.com/elastic/elastic-agent/issues/1069
+      timestamp: 1666095433
+      file:
+        name: 1666095433-service_runtime_v2.yaml
+        checksum: 94f777454bf571caf8da848441c43eb6d7b3bc10
+    - kind: feature
+      summary: Upgrade node to 18.12.0
+      description: ""
+      component: ""
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/1657
+      issue: []
+      timestamp: 1667243040
+      file:
+        name: 1667243040-Upgrade-node-to-18.12.0.yaml
+        checksum: 38b5473e0ee1165ad9c5d74ee969a9228c3fa68a
+    - kind: feature
+      summary: Add experimental support for running the elastic-agent-shipper
+      description: ""
+      component: ""
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/1527
+      issue:
+        - https://github.com/elastic/elastic-agent/issues/219
+      timestamp: 1667571017
+      file:
+        name: 1667571017-Add-support-for-running-the-elastic-agent-shipper.yaml
+        checksum: b2f5a0a8f9933c9ee624457116295aa674ef2a23
+    - kind: feature
+      summary: Add Common Expression Language input mapping to the Filebeat spec.
+      description: ""
+      component: spec
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/1719
+      issue: []
+      timestamp: 1668483030
+      file:
+        name: 1668483030-common-expression-language.yaml
+        checksum: 82427fc514372c067e140837177c8d81131ed424
+    - kind: feature
+      summary: Reimplement processes route
+      description: Re-implement processes route to restore compatibility with the current cloud health checks
+      component: ""
+      pr:
+        - https://github.com/elastic/elastic-agent/pull/1773
+      issue:
+        - https://github.com/elastic/elastic-agent/issues/1731
+      timestamp: 1669159455
+      file:
+        name: 1669159455-reimplement-processes-route.yaml
+        checksum: cbeb5b5766bf733dd5e200a69353e326c642a971
+    - kind: feature
+      summary: Capture stdout/stderr of all spawned components and adjust default log level to info for all components
```

Comment:
We have never defined what a component is anywhere in the agent documentation. I would just say sub-processes.

We should also mention that the logs for every sub-process except for Elastic defend are now combined into the Elastic agent log files.